<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电子秤检定系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="origin_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;原&lt;/p&gt;&lt;p&gt;始&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="detect_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>370</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;处&lt;/p&gt;&lt;p&gt;理&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="origin_image">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>591</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="detect_image">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>591</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>591</width>
      <height>50</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="detect">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Adobe Heiti Std</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>图片分割</string>
     </property>
    </widget>
    <widget class="QPushButton" name="image">
     <property name="font">
      <font>
       <family>Adobe Heiti Std</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>加载图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="model">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Adobe Heiti Std</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>加载模型</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="event_message">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>251</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="message_label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>21</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;事&lt;/p&gt;&lt;p&gt;件&lt;/p&gt;&lt;p&gt;信&lt;/p&gt;&lt;p&gt;息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="result_message">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>450</y>
      <width>21</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;识&lt;/p&gt;&lt;p&gt;别&lt;/p&gt;&lt;p&gt;结&lt;/p&gt;&lt;p&gt;果&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
